exercises/darts/canonical-data.json

```
@@ -1,13 +1,13 @@
 {
   "exercise": "darts",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "comments": [
-    "Return the correct amount earned by a dart landing in a given point in the target problem"
+    "Return the correct amount earned by a dart landing in a given point in the target."
   ],
   "cases": [
     {
       "property": "score",
-      "description": "A dart lands outside the target",
+      "description": "Dart missed target",
```

The "dart" bit has been removed from all the other cases, maybe also remove it here?

```suggestion
      "description": "Missed target",
```